projects/sbb-esta/angular-public/file-selector/src/_file-selector.scss

```diff
@@ -11,18 +11,30 @@
 
 @mixin fileSelectorButton() {
   @include buttons;
+  @include secondaryButton;
 
-  &:not(.sbb-file-selector-disabled) {
-    @include secondaryButton;
-    @include arrowbuttons;
+  @if $sbbBusiness {
+    .sbb-svgsprite-icon {
+      display: none;
+    }
 
+    &.sbb-file-selector-disabled {
+      background-color: $sbbColorMilk;
+      color: $sbbColorGraphite;
+      &:hover {
```

Comment:
Maybe you can group &.sbb-file-selector-disabled and hover state like `&.sbb-file-selector-disabled, &.sbb-file-selector-disabled:hover`